### PR TITLE
Add Grafana Labs as user

### PR DESCRIPTION
**Pull request by bboreham**
As requested at https://github.com/kedacore/keda/issues/3644#issuecomment-1245904704

cc @richih @JorTurFer

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -403,6 +403,10 @@ logo = "hmcts.png"
 url = "https://www.intercept.cloud"
 logo = "intercept.png"
 
+[[params.users]]
```
Comment (tomkerkhove) [inline config.toml:406]: Nit - Can we alphabetically sort please?

**Review by tomkerkhove — APPROVED** [review]
Whoa, thanks a ton!

File: config.toml (hunk 1 above)
Comment (bboreham) [reply config.toml]: Sorry!  I had it clearly in mind that the names were in alphabetical order, but evidently encountered turbulence on the way to landing the change.

Have updated and hope it is right now.

Comment (tomkerkhove) [reply config.toml]: No worries, that's what PRs are for!

**Comment by tomkerkhove** [comment]
@bboreham @richih How are you feeling about doing a end-user case study on how you use KEDA? I'm happy to get this done and show up on CNCF blog - If you have time?

**Comment by bboreham** [comment]
Excellent timing: https://grafana.com/blog/2022/10/20/how-to-autoscale-grafana-loki-queries-using-keda/?mdm=social

**Comment by JorTurFer** [comment]
Awesome!!! ❤️ 
BTW @bboreham , we have added a [Loki Scaler](https://github.com/kedacore/keda/pull/3700) from a community contribution, might you can take a look as experts you are?
If you want, you can help to maintain it, obviously only if you are willing, in other case the scaler will be maintained by community, but usually for the end-users, the trust in the company behind a product is higher than the trust in the community

**Comment by bboreham** [comment]
It's using Loki as a metrics source to feed into scaling decisions?

[this wasn't obvious to me from the PR; the issue 3699 is a tiny bit more explicit]

**Comment by JorTurFer** [comment]
Yes, that's it.
That PR enables Loki as a metrics source to scale based on Loki queries.

**Comment by tomkerkhove** [comment]
Nice, I'll definately read that one @bboreham! Let me know if you want to get this on CNCF & KEDA blog (cross-posted)

**Comment by RichiH** [comment]
It probably makes more sense to do an interview on top, Google er al
downrank both articles if it's a plain repost.
The article can inform questions, maybe?

Sent by mobile; please excuse my brevity.

**Comment by tomkerkhove** [comment]
@RichiH @bboreham We are looking for KEDA end-users who want to be interviewed by CNCF for our CNCF Graduation proposal.

Are you willing to share about your experiences on using KEDA? This can be anonymously if need be.

If so, feel free to fill in https://forms.gle/91dD4RVxifWsS2hc7

**Comment by tomkerkhove** [comment]
Hi @RichiH @bboreham; I wanted to follow-up to hear if you are willing to talk to our TOC sponsor?

**Comment by tomkerkhove** [comment]
As @RichiH knows how this goes =)

**Pull request merged**